explainer.md

```diff
@@ -1,3 +1,122 @@
 ## Explainer
 
-### Examples
+### Introduction
+
+This document explains [extensions to the WEBRTC specification](https://w3c.github.io/webrtc-extensions). This document contains a number of sections, each of which contains design details to one specific extension.
+
+### RTCRtpContributingSource extensions on captureTimestamp and senderCaptureTimeOffset
+
+Authors:
+minyue@chromium.org
+hbos@chromium.org
+chxg@google.com
+
+Participate
+https://www.chromestatus.com/feature/5728533701722112
+
+#### Abstract
+
+This section explains the two new data fields in RTCRtpContributingSource, namely captureTimestamp and senderCaptureTimeOffset, see [here](https://w3c.github.io/webrtc-extensions/#rtcrtpcontributingsource-dictionary). They are introduced for audio-video synchronization and end-to-end delay measurements.
+
+The solution to audio-video synchronization and end-to-end delay measurements described here is particularly desired by systems, where an intermediate steam regenerator that terminates the streams originating from senders, an audio mixer as an example, is involved.
```

Comment:
Nit: no comma after "systems".
More readable: "systems where an intermediate stream regenerator that terminates the streams originating from senders is involved. One example of this is an audio mixer."